src/design/theme-install.md

```diff
@@ -2,25 +2,23 @@
 title: Installing a New Theme
 ---
 
-When you first install Magento, the design elements of the store are based on the “Default” theme. The Luma theme is also available if you installed the sample data. You can modify a theme, add themes created by others, or create new ones. [Magento Marketplace]({% link magento/magento-marketplace.md %}) contains a growing selection of themes that you can install to change the appearance of your store. To learn how to create or modify a theme, see the [Frontend Developer Guide][1].
+When you first install Magento, the design elements of the store are based on the default _Magento Blank_ theme. The _Luma_ theme is also available if you installed the sample data. You can modify a theme, add themes created by others, or create new ones. [Magento Marketplace]({% link magento/magento-marketplace.md %}) contains a growing selection of themes that you can install to change the appearance of your store. To learn how to create or modify a theme, see our [Frontend Developer Guide][1].
 
 ![]({% link images/images/marketplace-themes.png %}){: .zoom}
 _Magento Marketplace_
 
-## To install a new theme:
-
 Before you begin, make sure that you have a valid set of [Magento 2 Access Keys][2]{:target="_blank"}.
 
-Follow the steps outlined in the [Marketplace Quick Tour][3], to:
+Follow the instructions available on the Marketplace to add a new theme to your Magento store:
 
-* Find a Theme
-* Evaluate the Product
-* Make a Purchase
-* Download the Software
-* Install the Theme
+1. Find a theme
+1. Evaluate the product
+1. Make a purchase
+1. Download the package
+1. [Install the theme][3]
 
-To apply the theme to your store, see: [Using the Default Theme]({% link design/theme-default.md %}).
+To apply the theme to your store, see [Using the Default Theme]({% link design/theme-default.md %}).
 
 [1]: http://devdocs.magento.com/guides/v2.3/frontend-dev-guide/bk-frontend-dev-guide.html
 [2]: https://devdocs.magento.com/guides/v2.3/install-gde/prereq/connect-auth.html
-[3]: http://docs.magento.com/marketplace/user_guide/quick-tour/welcome.html
+[3]: https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/theme-install.html
```

Comment:
the text refers to create or modify, but the link goes to "install third party theme".   recommend using the parent topic.   https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/themes/theme-overview.html

Reply:
will do